src/components/app/dtsiUserAvatar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { CSSProperties } from 'react'
+
+import { MaybeNextImgProps, NextImage } from '@/components/ui/image'
+import { DTSI_PublicUser } from '@/data/dtsi/generated'
+import { cn, twNoop } from '@/utils/web/cn'
+
+type AvatarBaseProps = {
+  size: number
+  className?: string
+  style?: CSSProperties
+}
+
+export type AvatarProps = (
+  | Omit<MaybeNextImgProps, 'width' | 'height'>
+  | { firstInitial: string; lastInitial?: string }
+) &
+  AvatarBaseProps
+
+const getFontSize = (size: number) => {
+  if (size < 20) {
+    return twNoop('text-sm')
+  }
+  if (size < 30) {
+    return twNoop('text-base')
+  }
+  return twNoop('text-lg')
+}
+
+const Avatar: React.FC<AvatarProps> = ({ size, className, style, ...props }) => {
+  if ('src' in props) {
+    return (
+      <div className={cn('relative', className)} style={{ width: size, height: size, ...style }}>
+        <NextImage
+          className={cn('rounded-full')}
+          fill
+          sizes={`${size}px`}
+          style={{ objectFit: 'cover' }}
```

Comment:
np: `NextImage` has a `objectFit` prop, not sure if they do anything else that pass it as a style but just in case

Reply:
I dont see in the docs :( How would you adjust?

Reply:
Well, I guess not having it in the docs means it's only a shorthand then. But you could do
```suggestion
          objectFit="cover"
```